### PR TITLE
Update dry-validation to avoid deprecation errors

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

File: config/workflows/mediated_deposit_workflow.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
                     "notifications": [
                         {
                             "notification_type": "email",
-                            "name": "Hyrax::Workflow::CompleteNotification",
+                            "name": "Hyrax::Workflow::DepositedNotification",
```
Comment (jeremyf) [inline config/workflows/mediated_deposit_workflow.json]: Why was this changed as part of the update for the dependency?

Comment (jcoyne) [inline config/workflows/mediated_deposit_workflow.json]: The hyrax version got updated, and this change was made in hyrax.
